packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/components/AddWalletButton/index.tsx

```diff
@@ -65,11 +65,30 @@ const AddWalletButton = ({ device, instances, addDeviceInstance, selectDeviceIns
         });
     };
 
+    type IsLockedDeviceWrapperProps = {
+        children: React.ReactElement;
+        isLocked: boolean;
+        wrapper: (children: React.ReactElement) => JSX.Element;
+    };
+
+    const IsLockedDeviceWrapper: React.FC<IsLockedDeviceWrapperProps> = ({
```

Comment:
Is there any reason not to use TooltipConditional instead of a new single-purpose component?